### PR TITLE
bug fix should use number not build when getting

### PR DESCRIPTION
**Pull request by alectronic0**
the version file hold 
version:
number:
commit:
date: 

not build:

fixed this

### CODE REVIEW

File: healthcheck/versionFile.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ const versionFile = () => {
 
 const version = () => {
   return versionFile().then(props => {
-    return (props.version) ? (props.build) ?  props.version + "-" + props.build : props.version : defaultObj.version
+    return (props.version) ? (props.number) ?  props.version + "-" + props.number : props.version : defaultObj.version
```
Comment (timja-kainos) [inline healthcheck/versionFile.js]: can you remind me where this is used?
i.e. I can't see a build or a number here:
https://www-dev.moneyclaim.reform.hmcts.net/health
this has a number (but from a different library):
https://www-dev.moneyclaim.reform.hmcts.net/info

**Comment by alectronic0** [comment]
https://git.reform.hmcts.net/reform/jenkins-library/blob/e5d5293d5e08200677a63d3bffc555f95e3247b7/src/uk/gov/hmcts/Versioner.groovy

we've used it in healthcheck to display the build number with version.

```
"buildInfo": {
"environment": "",
"project": "",
"name": ",
"version": "1.0.1-115",
"commit": "",
"date": "",
"extra": {}
}
```

looks like this has changed recently
https://git.reform.hmcts.net/reform/jenkins-library/commit/e5d5293d5e08200677a63d3bffc555f95e3247b7

**Comment by timja-kainos** [comment]
right, it has moved to here now:
https://github.com/hmcts/nodejs-info-provider/blob/master/src/versionFile.ts

1. do you want to migrate over?
2. fix this?
3. do both?

**Comment by alectronic0** [comment]
both.
DM still using it and we want to still build number on healthcheck till then.
this should fix healthcheck build info for now.

**Pull request merged**